lib/FIR/common.h

```diff
@@ -50,24 +50,13 @@ using FunctionType = evaluate::SomeType;  // TODO: what should this be?
 using AttributeList = std::vector<Attribute>;
 enum struct LinkageTypes { Public, Hidden, External };
 using Expression = evaluate::GenericExprWrapper;
```

Comment:
Why use `GenericExprWrapper` rather than `Expr<SomeType>`?

Reply:
No special reason. `GenericExprWrapper` was the class in the parse tree, so it was just going with the flow. On the other hand, my intention is to use `Expression` everywhere in FIR so that replacing these with the optimal class can be done seamlessly.